src/phoenix/trace/llama_index/callback.py

```diff
@@ -18,21 +17,43 @@
 
 
 def _check_instrumentation_compatibility() -> bool:
-    if find_spec("llama_index") is None:
-        raise PackageNotFoundError("Missing `llama-index`. Install with `pip install llama-index`.")
-    # split the version string into a tuple of integers
-    llama_index_version_str = version("llama-index")
-    llama_index_version = tuple(map(int, llama_index_version_str.split(".")[:3]))
+    try:
+        llama_index_version_str = version("llama-index")
+    except PackageNotFoundError:
+        llama_index_version_str = None
```

Comment:
```suggestion
    llama_index_version_str = None
    try:
        llama_index_version_str = version("llama-index")
    except PackageNotFoundError:
        ...
```
Is it a bit simpler this way arround?

Reply:
addressed in 944ce118